crates/bevy_sprite/src/mesh2d/mesh.rs

```diff
@@ -221,8 +221,6 @@ pub struct RenderMesh2dInstances(EntityHashMap<RenderMesh2dInstance>);
 pub struct Mesh2d;
 
 pub fn extract_mesh2d(
```

Comment:
What's with these changes? Unclear why at first glance what the original code was doing.